PathToRegex/PathToRegex.swift

```diff
@@ -20,7 +20,12 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
-import Regex
+
+#if COCOAPODS
+    import CrossroadRegex
```

Comment:
Sure!
 then I will just wait for you to release it. Ping me when its pushed so i can update